src/components/Header/LargeHeader.js

```diff
@@ -4,8 +4,8 @@ import styles from './LargeHeader.styles';
 import { Color } from 'lib/theme';
 import { ReactComponent as Logo } from 'static/icons/logo.svg';
 
-const LargeHeader = () => (
-  <div css={styles.root}>
+const LargeHeader = ({ ...rest }) => (
```

Comment:
In a perfect world we would avoid spreading props and using named props instead. Is that possible here?

eg: remove the `...rest` and replace with props
```
const LargeHeader = ({ propOne, propTwo, propThree }) => ()
```

Reply:
Actually it doesn't look like we're passing any props at the moment so we may be able to remove `...rest` and leave it empty??

Reply:
This has been removed, thanks @joshrounsville!